tasks/pre_install_checks.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: Gather facts on installed packages
+  package_facts:
+
```

Comment:
Please add `manager: rpm`

Reply:
and also `no_log: yes`

Reply:
I can add `manager: rpm` but then I might need to add additional conditional that checks that we are on RHEL-based system, right? Or is that not necessary?

Reply:
It is unnecessary.

Reply:
I'll add it I just wonder: why the `no_log` statement here? Is this considered sensitive?

Reply:
Both reasons.